plugins/fs/src/commands.rs

```diff
@@ -336,7 +336,7 @@ pub fn read_file<R: Runtime>(
     command_scope: CommandScope<Entry>,
     path: SafePathBuf,
     options: Option<BaseOptions>,
-) -> CommandResult<Vec<u8>> {
+) -> CommandResult<tauri::ipc::Response> {
```

Comment:
I think this is out of this PR scope, no? and there is a one or more functions that will need this change so better make a new PR

Reply:
probably committed by accident actually lol